demo/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
I think the demo should be as simple as possible -> `.gitignore` is not needed

Reply:
all files created by `create-react-app`. The only change is `App.js` and fixture with json.
It's a demo so I don't think we should change it.